Adds onboarding introduction page (#134)
BitcoinDesign/Guide: guide/onboarding/introduction.md

---
layout: guide
title: Onboarding
description: An overview of some of the concepts to consider when building onboarding experiences.
nav_order: 2
has_children: true

> permalink: /guide/onboarding/introduction/

Reviewer: Will you add a preview image later?

Author: Yeah I've updated this now, was waiting on some illustrations but I'll put something in for now and change later.

---

<!--

Editor's notes

A brief introduction and summary of all pages in this section. The idea is that readers
scan this page to get an overview of the section and then decide which topics to dive into.

-->

# Onboarding

> Effective onboarding can make or break someone’s first bitcoin experience. Therefore, it is important to identify your target audience when building wallet-products and design specifically with that audience in mind. 

Reviewer: ```suggestion
Onboarding, the first use, can make or break someone’s experience of a product. Therefore it is important to identify your target audience and design specifically with its context, expectations, and needs in mind. 
```

This is just a suggestion. Something doesn't flow quite right with the original text. One thing is that it's not only the Bitcoin experience, but the product experience overall. Also wanted to reduce the duplicate use of the word "audience" in the second sentence. Also wanted to clarify what onboarding is right away. Just a few ideas for you to consider.

Author: I prefer this actually, the use of wallet-product early on can be a bit confusing

> Some new users bring years of traditional financial experience to bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes them tricky to onboard. Users might assume that the patterns and habits from prior experience will apply, so convincing them to use a different financial system might be better in the product. 

Reviewer: ```suggestion
Some new users bring years of traditional financial experience to bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes them tricky to onboard, as they might assume that the patterns and habits from prior experience will apply. Convincing them to use a different financial system might be better in the product. 
```

Not totally clear what you mean by `might be better in the product.`. As opposed to what?

Author: Trying to get across the idea that the experience of using a bitcoin product should be self-evident that it is better than the traditional financial system.

> On the other hand, more than four billion people have no exposure to traditional financial institutions. This, along with designing for younger people whose ideas about money aren’t fully formed, represents a very different challenge.

Reviewer: Backing this up with research would be great. No worries if there is no good reference.

Reviewer 2: I would strike or rethink this phrase: "This makes them tricky to onboard."
Them, is basically every new user to bitcoin and this reads as them being unusual.

Author: The number seems to be closed to 2 Billion as well according to the World Bank - https://globalfindex.worldbank.org/sites/globalfindex/files/chapters/2017%20Findex%20full%20report_chapter2.pdf

Although I feel like I have definitely seen larger numbers. 

@danielnordh think I agree, will update 🚧

Author: Updated, let me know your thoughts

> Onboarding, for this guide, refers to anything that helps users set up the product or learn its interface (such as dedicated flows and UI elements). But onboarding isn’t just for first-time users. Experienced users also need to be onboarded, especially when new features are added. 

Reviewer: IIRC, you copy also focuses primarily on mobile wallets for consumers. If so, is that something worth pointing out?

> Remember: onboarding should not be a crutch for bad design. Avoid over indexing on trying to explain too much within onboarding, and then shooting users into a poorly designed app. Rather, think about onboarding flows as a smooth on-ramp into a great app experience.

Reviewer: ```suggestion
Remember: Onboarding should not be a crutch for bad design. Avoid trying to explain too much and then shooting users into a poorly designed app. Rather, think about onboarding flows as a smooth on-ramp into a great user experience.
```

Simplified a little bit. The short paragraph used the term "onboarding" three times which felt repetitive.

---

**Principles (coming soon)**

> Onboarding experiences can look very different depending on your target audience, however there are some things that should be consistent across bitcoin products.

Reviewer: ```suggestion
Onboarding experiences can look very different depending on your target audience, however there are some things that should be consistent across Bitcoin products.
```

---
layout: guide
title: Onboarding
description: An overview of some of the concepts to consider when building onboarding experiences.
nav_order: 2

> has_children: true

Reviewer: You can remove `has_children` if there are no sub-pages. With that property enabled, you can expand the section, even when there are no sub pages.

permalink: /guide/onboarding/introduction/
main_classes: -no-top-padding
image: /assets/images/guide/onboarding/onboarding.png
---

<!--

Editor's notes

A brief introduction and summary of all pages in this section. The idea is that readers
scan this page to get an overview of the section and then decide which topics to dive into.

-->

{% include picture.html
   image = "/assets/images/guide/onboarding/onboarding.png"
   retina = "/assets/images/guide/onboarding/onboarding@2x.png"
   mobile = "/assets/images/guide/onboarding/onboarding-mobile.png"
   mobileRetina = "/assets/images/guide/onboarding/onboarding-mobile@2x.png"
   alt-text = "Onboarding chapter illustration"
   width = 1600
   height = 600
   layout = "full-width"
%}

# Onboarding

Onboarding, the first use, can make or break someone’s experience of a product. Therefore it is important to identify your target audience and design specifically with its context, expectations, and needs in mind. 

> Some new users bring years of traditional financial experience to bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes onboarding tricky, as the user might assume that the patterns and habits from prior experience will apply. If you design a user experience that is superior to the traditional system, convincing users to swtich to a different financial system will be a lot easier. 

Reviewer: ```suggestion
Some new users bring years of traditional financial experience to Bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes onboarding tricky, as the user might assume that the patterns and habits from prior experience will apply. If you design a user experience that is superior to the traditional system, convincing users to swtich to a different financial system will be a lot easier. 
```

> Some new users bring years of traditional financial experience to Bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes onboarding tricky, as the user might assume that the patterns and habits from prior experience will apply. If you design a user experience that is superior to the traditional system, convincing users to swtich to a different financial system will be a lot easier. 

Reviewer 3: ```suggestion
Some new users bring years of traditional financial experience to Bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes onboarding tricky, as the user might assume that the patterns and habits from prior experience will apply. If you design a user experience that is superior to the traditional system, convincing users to switch to a different financial system will be a lot easier. 
```

On the other hand, close to [two billion](https://globalfindex.worldbank.org/sites/globalfindex/files/chapters/2017%20Findex%20full%20report_chapter2.pdf) people have no exposure to traditional financial institutions. This, along with designing for younger people whose ideas about money aren’t fully formed, represents a very different challenge.

Onboarding, for this guide, refers to anything that helps users set up the product or learn its interface (such as dedicated flows and UI elements) and initially focuses on mobile applications. But onboarding isn’t just for first-time users. Experienced users also need to be onboarded, especially when new features are added. 

Remember: Onboarding should not be a crutch for bad design. Avoid trying to explain too much and then shooting users into a poorly designed app. Rather, think about onboarding flows as a smooth on-ramp into a great user experience.

---

**Principles (coming soon)**

> Onboarding experiences can look very different depending on your target audience, however there are some things that should be consistent across Bitcoin products.

Reviewer 3: Maybe this sounds better?
```suggestion
Onboarding experiences can look very different depending on your target audience, however, some things should be consistent across Bitcoin products.
```
